src/com/fairphone/fplauncher3/Launcher.java

```diff
@@ -786,6 +788,12 @@ public void run() {
         } else if (requestCode == REQUEST_EDIT_FAVORITES) {
             mWaitingForResult = false;
             return;
+        } else if (requestCode == REQUEST_PREFERENCES) {
+            mModel.forceReload();
```

Comment:
It does not refresh the apps widget, so I'm not sure how to apply the new icon pack to the widget without restarting the launcher.